External/ChemDraw/CMakeLists.txt

```diff
@@ -2,114 +2,122 @@ add_custom_target(chemdraw_support ALL)
-  if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang" OR
+  if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang" OR
     CMAKE_CXX_COMPILER_ID STREQUAL "GNU" OR
     CMAKE_CXX_COMPILER_ID STREQUAL "Emscripten")
     # Require -fvisibility=hidden
     set_target_properties(ChemDraw PROPERTIES
-        CXX_VISIBILITY_PRESET hidden
-        VISIBILITY_INLINES_HIDDEN ON
+      CXX_VISIBILITY_PRESET hidden
+      VISIBILITY_INLINES_HIDDEN ON
     )
- endif()
+  endif()
 
+  # export all the symbols for ChemDraw on MSVC
+  if((MSVC AND BUILD_SHARED_LIBS) OR((NOT MSVC) AND WIN32))
+    set_target_properties(ChemDraw PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
+  endif()
 
-# export all the symbols for ChemDraw on MSVC                                                 
-if((MSVC AND BUILD_SHARED_LIBS) OR ((NOT MSVC) AND WIN32))                                
-    set_target_properties(ChemDraw PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS TRUE)                
-endif()                                                                                     
   # Suppress warnings since we don't control the chedraw code and indicate
-  #  we have the EXPAT_CONFIG created
+  # we have the EXPAT_CONFIG created
   if(MSVC)
-      ADD_DEFINITIONS("-DTARGET_API_LIB -D_WINDOWS -DTARGET_OS_WIN32 -DHAVE_EXPAT_CONFIG_H")
-      # we don't really control chemdraw source code, so suppress warnings	
-      target_compile_options(ChemDraw PRIVATE "/W0")
+    ADD_DEFINITIONS("-DTARGET_API_LIB -D_WINDOWS -DTARGET_OS_WIN32 -DHAVE_EXPAT_CONFIG_H")
+
+    # we don't really control chemdraw source code, so suppress warnings
+    target_compile_options(ChemDraw PRIVATE "/W0")
   else()
-      ADD_DEFINITIONS("-DTARGET_API_LIB -D__linux -DHAVE_EXPAT_CONFIG_H")
-      # we don't really control chemdraw source code, so suppress warnings
-      target_compile_options(ChemDraw PRIVATE -w -Wno-unknown-pragmas -Wno-error)
-      if(RDK_INSTALL_STATIC_LIBS)
-        if(TARGET ChemDraw_static)
-          target_compile_options(ChemDraw_static PRIVATE -w -Wno-unknown-pragmas -Wno-error)
-	endif()
-      endif(RDK_INSTALL_STATIC_LIBS)
+    ADD_DEFINITIONS("-DTARGET_API_LIB -D__linux -DHAVE_EXPAT_CONFIG_H")
+
+    # we don't really control chemdraw source code, so suppress warnings
+    target_compile_options(ChemDraw PRIVATE -w -Wno-unknown-pragmas -Wno-error)
+
+    if(RDK_INSTALL_STATIC_LIBS)
+      if(TARGET ChemDraw_static)
+        target_compile_options(ChemDraw_static PRIVATE -w -Wno-unknown-pragmas -Wno-error)
+      endif()
+    endif(RDK_INSTALL_STATIC_LIBS)
   endif()
 
   install(TARGETS ChemDraw DESTINATION ${RDKit_LibDir})
 
+  # When building static libs, expat will not be included into ChemDraw,
+  # so it needs to be installed so that it can be linked later on
+  # when the libs are used.
+  if(RDK_INSTALL_STATIC_LIBS OR NOT BUILD_SHARED_LIBS)
+    install(TARGETS expat EXPORT ${RDKit_EXPORTED_TARGETS} DESTINATION ${RDKit_LibDir} COMPONENT dev)
```

Comment:
Thanks for doing this, I initially just used the expat source directly so this wasn't an issue